### DISCUSSION TITLE
Dev poll: Ongoing support for Xcode 11?

### DISCUSSION THREAD

**Discussion by billhollings** [KhronosGroup/MoltenVK#1261] [General]
A few months ago, several devs identified the need to continue to support building with Xcode 11.7 for a while. But at some point, we'll need to sunset that support. So I'd like to get an idea of the need for this support.

At the time, the requests were for macOS targets.

1) Is support for building MoltenVK using Xcode 11.7 still needed for macOS targets (I know it is, but confirmation will help)?

2) Does anyone need to build MoltenVK for iOS targets using Xcode 11. 7?

**Comment by billhollings** [comment]
Investigating adding a Github Action for a macOS build using Xcode 11.7.

**Comment by Gcenx** [comment]
I don’t personally need this anymore but it would still be nice to have the support as it’s possible to still build MoltenVK on macOS Mojave using Xcode11.7 via Retroactive.

**Reply by iMonZ** [reply]
> I don’t personally need this anymore but it would still be nice to have the support as it’s possible to still build MoltenVK on macOS Mojave using Xcode11.7 via Retroactive.

Why aren't you upgrading to Catalina? The most people suffering from that are your users, if you don't use at least Xcode 12.4.
It would be great if developers could some day move forward

**Reply by Gcenx** [reply]
> Why aren't you upgrading to Catalina?

Because I still need native 32Bit support so macOS Mojave is the most ideal version, macports can also be coerced into building i386/x86_64 binaries.

> The most people suffering from that are your users, if you don't use at least Xcode 12.4.

Upgrading my Intel system to macOS Catalina or greater would make my users suffer.

> It would be great if developers could some day move forward

Until upstream wine has finished implementing 32on64 support there is no moving forward for my Intel system.

**Reply by iMonZ** [reply]
> > Why aren't you upgrading to Catalina?
> 
> Because I still need native 32Bit support so macOS Mojave is the most ideal version, macports can also be coerced into building i386/x86_64 binaries.
> 
> > The most people suffering from that are your users, if you don't use at least Xcode 12.4.
> 
> Upgrading my Intel system to macOS Catalina or greater would make my users suffer.
> 
> > It would be great if developers could some day move forward
> 
> Until upstream wine has finished implementing 32on64 support there is no moving forward for my Intel system.

You can use the father version of wine: https://media.codeweavers.com/pub/crossover/cxmac/demo/crossover-21.0.0.zip
This works pretty great on Catalina, BigSur and Monterey

**Reply by Gcenx** [reply]
> You can use the father version of wine: https://media.codeweavers.com/pub/crossover/cxmac/demo/crossover-21.0.0.zip
> This works pretty great on Catalina, BigSur and Monterey

Sigh, I'm talking about upstream wine here.......

I'm more than aware of CrossOver as I'm a BetterTester, here's something I created for the crossover-20.0.4 beta [Witcher 3 using Xbox One Controller macOS Big Sur 11.2](https://www.youtube.com/watch?v=ZPedXEMQNZU), this was before Apple fixed the controller API bug.

CodeWeavers wine requires a custom version of llvm/clang-8 to compile and still requires 32Bit libraries to support older versions of macOS like macOS Mojave. CrossOver-20.0.4/CrossOver-21.0.0 works on macOS High Sierra/Mojave/Catalina/Big Sur/Monterey Beta3 & Beta5.

Still doesn't make sense when I'm talking about upstream wine not CodeWeavers custom commercial version thats missing major components like gstreamer....
